04/index.js

```diff
@@ -1,29 +1,24 @@
-
-const ul = document.getElementById('js-lists');
+const ul = document.getElementById("js-lists");
 const fragment = document.createDocumentFragment();
 
-const contents = [
-  { href: 'a1.html', src: '/img/bookmark.png', text: 'a1' },
-  { href: 'a2.html', src: '/img/message.png', text: 'a2' }
+const contents =   [
+    { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+    { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" },
 ];
 
-for(let i = 0; i < contents.length; i++){
-  const image = document.createElement('img');
-  const anchor = document.createElement('a');
-  const li = document.createElement('li');
-
-  const content = contents[i];
-
-  image.src = content.src;
-  anchor.href = content.href;
-
-  li
-  .appendChild(anchor)
-  .appendChild(image);
+contents.forEach ((content)=> {
+  let image = document.createElement('img');
+  let anchor = document.createElement('a');
+  let li = document.createElement('li');
 
-  anchor.insertAdjacentHTML('beforeend',content.text);
+  image.src = content.img;
+  image.alt = content.alt;
+  anchor.href = "/" + content.to;
 
-  fragment.appendChild(li);
-}
+  fragment.appendChild(li)
+          .appendChild(anchor)
+          .appendChild(image)
+          .insertAdjacentText('afterend',content.text);
```

Comment:
```suggestion
　　 anchor.textContent = content.text
         fragment.appendChild(li).appendChild(anchor).insertAdjacentElement('afterbegin',img);
```
insertAdjacentText使ったことなかったです！勉強になりました！

このような書き方もできます :)
単なるコメントです！

Reply:
insertAdjacentTextですが、[課題3](https://github.com/sunacodesu/frontend-handson/pull/4#issuecomment-2267026880)で、長谷川さんが教えてくださって、早速利用してみました😊

書き方の種類、ありがとうございます！
「はるさんに頂いた書き方」として、コメントですが追加コミットしました！
2ef63a92f7c2b6fad45693d78cd2934bfc6e28fb

Reply:
わざわざありがとうございます！
次回からコメント追加しなくて大丈夫です🙆‍♀️